### PR TITLE
Update Finding  Good First Issues

### PR DESCRIPTION
**Pull request by Sahilvk**
These are typically simple, well-defined tasks that are ideal for newcomers to the project or those who want to contribute but are unsure where to start. Good first issues help to lower the barrier to entry for newcomers and allow them to make meaningful contributions to the project without feeling overwhelmed or intimidated. Ultimately, adding good first issues and providing clear instructions can help to make the project more accessible, more collaborative, and more successful in the long run.

- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by Sahilvk** [comment]
please consider my suggestion,if u think it will add value to guide

**Comment by tp2kiss05** [comment]
> These are typically simple, well-defined tasks that are ideal for newcomers to the project or those who want to contribute but are unsure where to start. Good first issues help to lower the barrier to entry for newcomers and allow them to make meaningful contributions to the project without feeling overwhelmed or intimidated. Ultimately, adding good first issues and providing clear instructions can help to make the project more accessible, more collaborative, and more successful in the long run.
> 
> * [x]  Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
> * [x]  Have you explained what your changes do, and why they add value to the Guides?
> 
> **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

**Comment by tp2kiss05** [comment]
[Brow&app](https://github.com/github/opensource.guide/pull/3004#issuecomment-1475259876)

**Comment by tp2kiss05** [comment]
Required

**Review by Soheilwikey — COMMENTED** [review]
@Soheilwikey
- نام: راه اندازی جاوا JDK
  موارد استفاده: actions/setup-java@v3.10.0

**Comment by atifnimran** [comment]
Okay

**Review by amirgreen18 — COMMENTED** [review]
https://github.com/github/opensource.guide/pull/3004#issue-1630818598 
شبکه باز جدید جهت تست

**Comment by ShehzadDev** [comment]
🙌

File: _articles/how-to-contribute.md [hunk 1]
```diff
@@ -370,6 +370,20 @@ A project that is friendly and welcoming signals that they will be receptive to
 — @kfogel, [_Producing OSS_](https://producingoss.com/en/evaluating-oss-projects.html)
   </p>
 </aside>
+## Finding Good First Issues
```
Comment (1234567gus) [inline _articles/how-to-contribute.md:373]: M

File: _articles/how-to-contribute.md [hunk 2]
```diff
@@ -370,6 +370,20 @@ A project that is friendly and welcoming signals that they will be receptive to
 — @kfogel, [_Producing OSS_](https://producingoss.com/en/evaluating-oss-projects.html)
```
Comment (1234567gus) [inline _articles/how-to-contribute.md:370]: > 
```suggestion
— @kfogel, [_Producing OSS_](https://producingoss.com/en/evaluating-oss-projects.html)

```

**Review by 1234567gus — COMMENTED** [review]
#
Duplicate of #

**Review by Anemia88 — COMMENTED** [review]
Genial

**Review by AnA025 — COMMENTED** [review]
https://github.com/github/opensource.guide/blob/4952cc12859e5864ab0827a072de8e9892db3bf9/_articles/how-to-contribute.md

**Review by AnA025 — COMMENTED** [review]
-to-contribute.md

**Review by Tbo7 — COMMENTED** [review]
open sorce( 
 New )

**Comment by nanis94** [comment]
comment

**Comment by Anemia88** [comment]
Genial implementar

**Review by Anemia88 — COMMENTED** [review]
Genial

File: _articles/how-to-contribute.md (hunk 2 above)
Comment (Sreeammuz369) [reply _articles/how-to-contribute.md:370]: > >

**Comment by adeliyamelwin666** [comment]
Fiduciary

**Review by limck301 — COMMENTED** [review]
https://github.com/github/opensource.guide/pull/3004#issuecomment-1475261999

**Comment by zkoppert** [comment]
Thanks for the PR, but we're going to pass on this for now.

**Pull request closed without merging**